transcendence-app/src/chat/infrastructure/db/postgres/chatmember.postgres.repository.ts

```diff
@@ -13,4 +18,31 @@ export class ChatMemberPostgresRepository
   constructor(protected pool: PostgresPool) {
     super(pool, table.CHATMEMBERS);
   }
+
+  async retrieveChatRoomMembers(
```

Comment:
suggestion (non-blocking): What do you think about adding pagination?

Reply:
A chat with many more than 20 people looks impractical to me. I thought about pagination, but I didn't think it was very useful to add it here

Reply:
I see your your point, and maybe 20 user for a private chat with your friends is ok, but for a public chat, 20 users seems to low for me. How many people are in the 42 slack channels? Hundreds?

Reply:
Will do, then.

Reply:
I've created a ticket for this.